tests/test_root_model.py

```diff
@@ -401,3 +401,20 @@ class Model(BaseModel):
 
     m = Model()
     assert m.value.root == 42
+
+
+def test_root_model_json_schema_meta():
+    ParametrizedModel = RootModel[int]
```

Comment:
Can this be done?
```suggestion
    ParametrizedModel = Annotated[RootModel[int], Field(description="potato")]
```

Reply:
Actually, can we add a test for this? 🤔

Reply:
Good idea! Wilco.

Reply:
This doesn't work actually. And I don't see how it could work. As far as I understand, we don't have control over what is outside RootModel here.

Reply:
I mean, does it work inside a model?

```py
class Model(BaseModel):
   a: Annotated[RootModel[int], Field(description="potato")]
```

Reply:
This is not a blocker for this PR btw, just wondering.

Reply:
Well, it works for RootModel when it acts as a field like it would for any field and it doesn't work when used on a RootModel when it's a final class like it wouldn't on any other model class.